client-ui/src/app/components/lineage/lineage-details/operation-properties-details/properties/properties.component.ts

```diff
@@ -41,7 +41,9 @@ export class PropertiesComponent implements OnDestroy {
   public propertyName: string
 
   @Input()
-  public properties: Property[]
+  public properties: Property[];
+
+  PropertyType = PropertyType;
```

Comment:
nit: we don't use semicolons in JS/TS code

Reply:
So, you are doing wrong. This is not Python and not JS, but Typescript. Have a look at any project, Angular, Angular CDK, React, Vu.js source code and you will see semicolumns everywhere.

Reply:
Here is what Sonar source say:

> 
**Statements should end with semicolons**
In JavaScript, the semicolon (;) is optional as a statement separator, but omitting semicolons can be confusing, and lead to unexpected results because a semicolon is implicitly inserted at the end of each line.

>

Reply:
There are long-long-long and never ending debates about automatic semicolon inference and its pros and cons. Simply saying that ins't _wrong_ just because Angular or Vue folks are doing it in another way is not wise. We could have have this discussion later and we might change it in the future, but for now consistency is more important. Let's discuss it offline.